pvlib/pvsystem.py

```diff
@@ -142,19 +142,18 @@ class PVSystem:
 
     module : string, optional
         The model name of the modules.
-        May be used to look up the module_parameters dictionary
-        via some other method.
 
-    module_type : string, default 'glass_polymer'
-         Describes the module's construction. Valid strings are 'glass_polymer'
-         and 'glass_glass'. Used for cell and module temperature calculations.
+    module_type : string, default `glass_polymer`
+         Describes the module's construction. Valid strings are `glass_polymer`
+         and `glass_glass`. Used for cell and module temperature calculations.
 
     module_parameters : dict or Series, optional
         Module parameters as defined by the SAPM, CEC, or other.
 
     temperature_model_parameters : dict or Series, optional
         Temperature model parameters as required by one of the models in
-        pvlib.temperature (excluding poa_global, temp_air and wind_speed).
+        :py:mod:`pvlib.temperature` (excluding ``poa_global``, ``temp_air`` and
+                                     ``wind_speed``).
```

Comment:
```suggestion
        :py:mod:`pvlib.temperature` (excluding ``poa_global``, ``temp_air`` and
        ``wind_speed``).
```

The indented version doesn't render as intended